dnf-plugins-core.spec

```diff
@@ -138,6 +138,9 @@ reposync commands. Additionally provides generate_completion_cache passive plugi
 
 %package -n dnf-utils
 Conflicts:      yum-utils < 1.1.31-513
+%if 0%{?rhel} != 7
```

Comment:
this is also true for Fedora, so I don't think that this is correct.

Reply:
It was actually meant that way.
AFAIK most of the python2 based software is going away in Fedora,
we'd also like to remove yum3 stack from Fedora 29.